addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-cidr-service.js

```diff
@@ -42,6 +46,10 @@ class EnvironmentScCidrService extends Service {
   }
 
   checkRequest(updateRequest) {
+    const isAppStreamEnabled = this.settings.optionalBoolean(settingKeys.isAppStreamEnabled, false);
+    if (isAppStreamEnabled) {
+      throw this.boom.badRequest(`CIDR operation unavailable when AppStream is enabled`, true);
```

Comment:
optional: CIDR _edit_ operation...

Reply:
This checkRequest can expand to other methods that can be potentially added in future. So I didn't add `edit` explicitly.